src/SFA.DAS.AdminService.Web/Services/Gateway/GatewayCriminalComplianceChecksOrchestrator.cs

```diff
@@ -23,24 +23,28 @@ public class GatewayCriminalComplianceChecksOrchestrator : IGatewayCriminalCompl
         }
 
         public async Task<OrganisationCriminalCompliancePageViewModel> GetCriminalComplianceCheckViewModel(GetCriminalComplianceCheckRequest request)
-        {          
-            var commonDetails = await _applyApiClient.GetPageCommonDetails(request.ApplicationId, request.PageId, request.UserName);
-
-            var model = new OrganisationCriminalCompliancePageViewModel 
-            { 
-                ApplicationId = request.ApplicationId, 
-                PageId = request.PageId,
-                ApplyLegalName = commonDetails.LegalName,
-                Ukprn = commonDetails.Ukprn,
-                Status = commonDetails.Status,
-                OptionPassText = commonDetails.OptionPassText,
-                OptionFailText = commonDetails.OptionFailText,
-                OptionInProgressText = commonDetails.OptionInProgressText,
-                GatewayReviewStatus = commonDetails.GatewayReviewStatus
-            };
+        {
+            //var commonDetails = await _applyApiClient.GetPageCommonDetails(request.ApplicationId, request.PageId, request.UserName);
 
+            //var model = new OrganisationCriminalCompliancePageViewModel 
+            //{ 
+            //    ApplicationId = request.ApplicationId, 
+            //    PageId = request.PageId,
+            //    ApplyLegalName = commonDetails.LegalName,
+            //    Ukprn = commonDetails.Ukprn,
+            //    Status = commonDetails.Status,
+            //    OptionPassText = commonDetails.OptionPassText,
+            //    OptionFailText = commonDetails.OptionFailText,
+            //    OptionInProgressText = commonDetails.OptionInProgressText
+            //};
```

Comment:
If not needed can it be deleted rather than commented out?

Reply:
done